README.md

```diff
@@ -404,16 +425,19 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 * [Apache Nutch](http://nutch.apache.org/) - Highly extensible, highly scalable Web crawler for production environment.
 * [Crawler4j](https://code.google.com/p/crawler4j/) - Simple lightweight alternative.
+* [Jerry](http://jodd.org/doc/jerry/index.html) - jQuery in Java.
 * [JSoup](http://jsoup.org/) - Scrapes, parses, manipulates and cleans HTML.
+* [Lagarto](http://jodd.org/doc/lagarto/index.html) - All purpose (X)HTML parser.
 
 ## Web Frameworks
 
 *Frameworks that handle the communication between the layers of an web application.*
-h
+
 * [Apache Tapestry](http://tapestry.apache.org/) - Component oriented framework for creating dynamic, robust, highly scalable web applications in Java.
 * [Apache Wicket](http://wicket.apache.org/) - Component-based web application framework similar to Tapestry with a stateful GUI.
 * [Google Web Toolkit](http://www.gwtproject.org/) - Toolbox which includes a Java-to-JavaScript compiler for client-side code, XML parser, API for RPC, JUnit integration, internationalization support and widgets for the GUI.
 * [Grails](https://grails.org/) - Groovy framework with the aim to provide a highly productive environment by favoring convention over configuration, no XML and support for mixins.
+* [Madvoc](http://jodd.org/doc/madvoc/index.html) - Fully-featured micro MVC web framework.
```

Comment:
Why should we add another web framework. I really don't understand how it is different and at what it excels.